### PR TITLE
Fix TTL cache issue

### PR DESCRIPTION
**Pull request by fabiocicerchia**
Ref #478, #410, #404

### CODE REVIEW

**Comment by JellyBellyDev** [comment]
Can you fix the tests please?

File: src/Contributors/Model/Contributor.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function getProfileUrl(): string
 
     public function getProfileImg(): string
     {
-        return $this->profileImg.'?s='.$this->size;
+        return $this->profileImg;
```
Comment (AlessandroMinoccheri) [inline src/Contributors/Model/Contributor.php:42]: Is it not necessary?

Comment (fabiocicerchia) [reply src/Contributors/Model/Contributor.php:42]: It's a known issue, for now it is a workaround.

**Review by AlessandroMinoccheri — APPROVED** [review]

**Review by garak — APPROVED** [review]

**Review by JellyBellyDev — APPROVED** [review]
LGTM

**Pull request merged**